pom.xml

```diff
@@ -189,22 +189,12 @@
       <groupId>com.atlassian.jira</groupId>
       <artifactId>jira-rest-java-client-api</artifactId>
       <version>${jira-rest-client.version}</version>
-      <exclusions>
```

Comment:
Are you sure those exclusions can be safely removed? (see comments in pom changes of #213 )

Reply:
@olamy I think you had some doubts about guava dependency...

Reply:
I'll double-check.
The best way forward would probably to focus the client update in #213 and not duplicate the work here.